### DISCUSSION TITLE
How do I fuzzy match elixir package version

### DISCUSSION THREAD

**Discussion by Pilgrim1379** [jdx/mise#607] [Q&A]
Elixir lists its package with versions like below:
```
1.14.5
1.14.5-otp-23
1.14.5-otp-24
1.14.5-otp-25
1.14.5-otp-26
```
If I always want rtx to upgrade/install the latest version of `1.14.x-otp-25`, where I don't mind what value `x` is, how do I specify this in my `config.toml`?

**Comment by jdx** [comment]
I'm not sure how we could support this unfortunately. rtx is based on prefixes for figuring out fuzzy versions
